tools/src/main/java/org/bitcoinj/tools/BuildCheckpoints.java

```diff
@@ -211,7 +211,16 @@ private static void writeTextualCheckpoints(TreeMap<Integer, StoredBlock> checkp
     }
 
     private static void sanityCheck(File file, int expectedSize) throws IOException {
-        CheckpointManager manager = new CheckpointManager(params, new FileInputStream(file));
+    	FileInputStream fis = new FileInputStream(file);
+        CheckpointManager manager;
+
+        try {
+            manager = new CheckpointManager(params, fis);
+        } catch (IOException ioe){
+            fis.close();
```

Comment:
ugh - weirdly I thought try-finally wasn't introduced until after Java 6, but that's of course try-with-resources. Ignore me. Will fix.